lib/csp.js

```diff
@@ -31,4 +33,29 @@ module.exports = function (options) {
         res.header(name, value);
         next();
     };
-};
\ No newline at end of file
+};
+
+var createPolicyString = module.exports.createPolicyString = function (policy) {
+    var entries;
+
+    if (thing.isString(policy)) {
+        return policy;
+    }
+
+    if (thing.isArray(policy)) {
+        return policy.map(createPolicyString).join('; ');
+    }
+
+    if (thing.isObject(policy)) {
+        entries = Object.keys(policy).map(function (directive) {
+            if (policy[directive] === 0 || policy[directive]) {
```

Comment:
I like checking for emptiness by doing `policy[directive] != null` (note the single `=`). This returns true iff `policy[directive]` is not `null` or `undefined`.